Shared/BundleExtensions.swift

```diff
@@ -8,6 +8,10 @@ extension Bundle {
     /// Get a string from the info plist dictionary for a given key
     public func infoDictionaryString(forKey key: String) -> String {
         guard let value = object(forInfoDictionaryKey: key) as? String else {
+            if ProcessInfo.processInfo.environment["XCTestConfigurationFilePath"] != nil {
```

Comment:
Perhaps we should wrap this somehow so its not executed at all on production builds?

Reply:
We don't do that for other similar cases, This deserves a separate ticket.
That would require us to add a new swift flag only to `DataTests` I think

I will change this line to use `AppConstants.IsRunningTest` though